adoc/admin-updates.adoc

```diff
@@ -17,6 +17,23 @@ Run `sudo zypper update` on the management workstation before any attempt to upd
 
 Updating of {kube} components is handled via `skuba`.
 
+The general procedure should look like this:
+
+. Update the packages and reboot your management workstation to get all the latest changes to skuba and it's dependencies
+. Run `skuba addon upgrade apply` on the management workstation
+.
+. Check if there are addon upgrades available for the current cluster
+. Upgrade all master nodes
+. Upgrade all worker nodes
+. Check if new addons are available for the new version
+.. If necessary upgrade master nodes again
+.. If necessary upgrade worker nodes again
```

Comment:
If necessary upgrade addons again
and re-apply config changes again

Reply:
After all of this update helm / tiller
Then update all helm deployments and other deployments in the cluster.